api.js

```diff
@@ -195,7 +195,7 @@ function handlePaths(files) {
 		files = [
 			'test.js',
 			'test-*.js',
-			'test/*.js'
```

Comment:
I think what you are doing here is inconsistent and confusing.

The default is currently the same as running:

```
ava test.js test-*.js test/*.js
```

If we don't make this change that stays consistent. To engage the "recursive directory behavior" they should need explicitly provide a directory name they want recursed. There is less chance of an error that way. I find it far more confusing that there are `.js` globs for the base-dir, but we will recurse the test folder indefinitely.